src/steps/check-lighthouse-performance.ts

```diff
@@ -25,33 +25,81 @@ export class CheckLighthousePerformance extends BaseStep implements StepInterfac
     try {
       const currentUrl = await this.client.getCurrentPageInfo('url');
       const lhr = await this.client.getLighthouseScores(currentUrl.toString(), throttleTo);
-
       const performance: any = Object.values(lhr.categories)[0];
       const actualScore = Math.round(performance.score * 100);
       if (actualScore < expectedScore) {
         const audits = Object.values(lhr.audits);
-        return this.fail('The page\'s performance score of %d was lower than the expected score of %d in %s.\n\n\n Opportunities for improvement:\n%s', [
+        const recordTable = this.createFailRecord(audits);
+        const recordKeyValue = this.createSuccessRecord(lhr);
+
+        return this.fail(
+          'The page\'s performance score of %d was lower than the expected score of %d in %s.\n\n\n Opportunities for improvement:\n%s',
```

Comment:
- [ ] Now that the table is visible here, we can remove the `Opportunities for improvement:\n%s` portion of this message.